Description: allow freeplane to access code from /usr/share/knopflerfish/*,
  /usr/share/java/* and /usr/share/maven-repo/**
Author: Felix Natter <fnatter@gmx.net>
Forwarded: not-needed
Last-Update: 2016-05-05
--- a/freeplane_framework/script/freeplane.policy
+++ b/freeplane_framework/script/freeplane.policy
@@ -2,6 +2,18 @@
         permission java.security.AllPermission;
 };
 
+grant codeBase "file:/usr/share/knopflerfish/*" {
+        permission java.security.AllPermission;
+};
+
+grant codeBase "file:/usr/share/java/*" {
+        permission java.security.AllPermission;
+};
+
+grant codeBase "file:/usr/share/maven-repo/-" {
+        permission java.security.AllPermission;
+};
+
 grant {
         permission java.lang.RuntimePermission "getClassLoader";
 };
